src/kaocha/type/var.clj

```diff
@@ -17,28 +17,32 @@
   (println "Test ran without assertions. Did you forget an (is ...)?")
   (report/print-output m))
 
-(defmethod testable/-run :kaocha.type/var [{test    :kaocha.var/test
-                                            wrap    :kaocha.testable/wrap
-                                            the-var :kaocha.var/var
-                                            meta'   :kaocha.testable/meta
-                                            :as     testable} test-plan]
+(defn test-var [{test    :kaocha.var/test
```

Comment:
Haha if you look at the commits, you'll see i went back and forth on that. I went with wrapping with the function at the call site instead of returning a function because 1) it matches how clojure.test does it and 2) I find it harder to reason about functions that return functions vs letting the wrapping function be part of the implantation details.

However, both are reasonable and I'm good to change it if you desire.

Reply:
Those reasons make sense. The reason for returning a function is that I'm not sure you'd ever run `test-var` *without* wrapping it. 

Maybe it just needs a clearer name, like `run-test-var`. I guess the `test` in `test-var` could already be interpreted as a verb, but it's not clear to me.